nuxeo-runtime/nuxeo-runtime-test/src/test/java/org/nuxeo/runtime/metrics/TestMetricsService.java

```diff
@@ -18,33 +18,56 @@
  *
  */
 
-package org.nuxeo.runtime;
+package org.nuxeo.runtime.metrics;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+import static org.junit.Assert.assertNotNull;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import javax.inject.Inject;
 
 import org.junit.Test;
 import org.junit.runner.RunWith;
-import org.nuxeo.runtime.metrics.MetricsDescriptor;
+import org.nuxeo.runtime.test.runner.Deploy;
 import org.nuxeo.runtime.test.runner.Features;
 import org.nuxeo.runtime.test.runner.FeaturesRunner;
 import org.nuxeo.runtime.test.runner.RuntimeFeature;
 
-import java.util.Arrays;
-import java.util.List;
-import java.util.stream.Collectors;
-
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertNotEquals;
+import io.dropwizard.metrics5.MetricName;
 
 /**
- * @since 9.3
+ * @since 11.1
  */
 @RunWith(FeaturesRunner.class)
 @Features(RuntimeFeature.class)
-public class TestGraphiteReporter {
+@Deploy("org.nuxeo.runtime.metrics")
+@Deploy("org.nuxeo.runtime.metrics:test-metrics-contrib.xml")
+public class TestMetricsService {
+
+    @Inject
+    protected MetricsService service;
 
     @Test
-    public void testFilterMetrics() {
+    public void testService() {
+        assertNotNull(service);
+        service.stopReporters();
+        service.startReporters();
+        service.stopReporters();
+        service.startReporters();
```

Comment:
What is the wish to start/stop reporters several times?

Reply:
this is exposed by an automation endpoint better to check that they can be started/stopped multiple time without problem

Reply:
Worth adding a small comment maybe? So that someone won't clean it up at some point 😬